packages/utils/src/cli.ts

```diff
@@ -405,6 +405,15 @@ export function readTSConfig<T>(configPath: string): T {
   return json5.parse(readFileSync(configPath, 'utf-8'));
 }
 
+/**
+ * Finds the closest ancestor to `startPath` directory containing package.json file
+ */
+export function findPackageRootDirectory(startPath: string): string | undefined {
+  const packageJSONPath = findupSync('package.json', { cwd: startPath });
+
+  return packageJSONPath ? parse(packageJSONPath).dir : undefined;
+}
+
```

Comment:
Refactor ember-app-project-graph to use this method.

Also had a stopping point.

```
    // Findup till we find package.json relative to the dir of the entrypoint.
    const foundPackageJson = findUpSync('package.json', {
      cwd: lookUpDir,
      stopAt: this.rootDir,
    });
```

Reply:
Will do that, thanks.